.github/workflows/verify_artifacts.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import argparse
+
+
+def create_artifact_paths(version):
+    artifacts = []
+    artifacts_base_path = "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu"
+    # add to this list here to update the list of artifacts to check 
+    besu_paths = [
+        f"evm/{version}/evm-{version}",
+        f"plugin-api/{version}/plugin-api-{version}",
+        f"internal/metrics-core/{version}/metrics-core-{version}",
+        f"internal/core/{version}/core-{version}",
+        f"internal/config/{version}/config-{version}"
+    ]
+    for path in besu_paths:
+        artifacts.append(f"{artifacts_base_path}/{path}.module")
+        artifacts.append(f"{artifacts_base_path}/{path}.pom")
+        artifacts.append(f"{artifacts_base_path}/{path}.jar")
+    return artifacts
```

Comment:
also verify bom is present

Reply:
bom module has .pom and .module  eg https://hyperledger.jfrog.io/ui/native/besu-maven/org/hyperledger/besu/bom/24.12.2/

Reply:
👍 Done @macfarla. This is now a plain list so easy to edit. Ready for review